src/Language/PureScript/Docs/Convert/Single.hs

```diff
@@ -135,10 +135,10 @@ basicDeclaration :: String -> DeclarationInfo -> Maybe IntermediateDeclaration
 basicDeclaration title info = Just $ Right $ mkDeclaration title info
 
 convertDeclaration :: P.Declaration -> String -> Maybe IntermediateDeclaration
-convertDeclaration (P.TypeDeclaration _ ty) title =
-  basicDeclaration title (ValueDeclaration ty)
-convertDeclaration (P.ExternDeclaration _ ty) title =
```

Comment:
I had assumed that explicit type declarations, where given, would be left alone by the compiler, but I will check this. But I think I've found another bug in the meantime...